sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -329,7 +329,9 @@ internal static PiiTask ConvertToPiiTask(RecognizePiiEntitiesOptions option)
                 {
                     Domain = option.DomainFilter.HasValue ? option.DomainFilter.Value.GetString() : (PiiTaskParametersDomain?)null,
                     ModelVersion = !string.IsNullOrEmpty(option.ModelVersion) ? option.ModelVersion : "latest",
-                    StringIndexType = option.StringIndexType
+                    StringIndexType = option.StringIndexType,
+                    LoggingOptOut = option.DisableServiceLogs.HasValue ? option.DisableServiceLogs : null
+                    // Categories are not enabled becuase of https://github.com/Azure/azure-sdk-for-net/issues/19237
```

Comment:
Is this still true?

Reply:
As for what we have in westus2 yes. need to try in the new deployment, but for now, putting the comment as I forgot why we didn't have it, and took me a while to understand it was intentional and not a bug in .NET